test/conference.js

```diff
@@ -336,24 +287,16 @@ contract('Conference', function(accounts) {
       await conference.ended().should.eventually.eq(true)
     })
 
-    it('cannot withdraw if non owner calls', async function(){
+    it('rejects if non owner calls', async function(){
       await conference.payback({from:non_owner}).should.be.rejected;
       await conference.withdraw({from:attended}).should.be.rejected;
       // money is still left on contract
       await getBalance(conference.address).should.eventually.eq( mulBN(deposit, 2) )
       await conference.isPaid(attended).should.eventually.eq(false)
     })
 
-    it('cannot withdraw if you did not attend', async function(){
```

Comment:
Do you have equivalent test of checking you cannot withdraw if haven't finalised yet?

Reply:
Added